src/libs/actions/Report.js

```diff
@@ -728,7 +729,7 @@ function getReportChannelName(reportID) {
  * @param {Boolean} isChunked
  */
 function subscribeToPrivateUserChannelEvent(eventName, onEvent, isChunked = false) {
-    const pusherChannelName = `private-user-accountID-${currentUserAccountID}`;
+    const pusherChannelName = `private-user-accountID-${currentUserAccountID}${CONFIG.PUSHER.SUFFIX}`;
```

Comment:
In the near future we're going to encrypt the user channel (`private-encrypted`) but the report channel will remain `private`. I like the idea of adding `${CONFIG.PUSHER.SUFFIX}` in one place instead of all these files, maybe a different function name like `Pusher.prepareChannelName` or something generic just about the channel name?

Reply:
yeah, I'm just not sure where to put it, should it be part of Pusher or not 
For example there's `Pusher.init` where we pass values from `CONFIG.js` so it seems importing `CONFIG.js` directly in Pusher was deliberately avoided

But if we are to create a method like `Pusher.prepareChannelName` we'll have to import `CONFIG.js` to pusher

Reply:
True true, looks like @tgolen made those changes, though it was a long time ago. I can't see why we wouldn't want to import Config into Pusher since we import Config into lots of other places 🤔

Reply:
I don't think there is much harm in importing `CONFIG.js`. I think it was at one point helpful to have it with fewer dependencies because some of this code I think originally came from OldDot. I would still be in favor of keeping it low on dependencies, and then having a method for `getChannelName()` which can add the suffix if it needs to.

Reply:
Ok so it sounds like you're good to add the new method to `Pusher` 👍